hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcCall.java

```diff
@@ -83,7 +86,17 @@ public interface RpcCall extends RpcCallContext {
   /** Returns The request header of this call. */
   RequestHeader getHeader();
 
-  ConnectionHeader getConnectionHeader();
+  /**
+   * Returns the map of attributes specified when building the Connection See the Map argument on
```

Comment:
nitpick: since you already have to do one more revision can you fix this and below javadoc to at least add a period prior to `See the..`? Could also add a `<br>` if you want them to be on the next line, or better yet use `@see`, i.e.:

```java
/**
 * Returns the map of attributes specified when building the Connection.
 * @see ConnectionFactory#createConnection(Configuration, ExecutorService, User, Map)  
 */
```
The see will automatically also do the linking.